Domain.Sql/Migrations/AzureSqlDbMigrator.cs

```diff
@@ -94,12 +95,36 @@ public MigrationResult Migrate(DbContext context)
             var sql =
                 $@"
 alter database {context.Database.Connection.Database} 
-modify (MAXSIZE = {MaxSize},
+modify (MAXSIZE = {MaxSize}MB,
```

Comment:
You can express it in MB. 10GB = 10 * 1024. I don't like strings. I even thought of making the editions and service levels an enum but Azure do keep changing their SKU's.